tensorflow/lite/java/demo/app/src/main/java/com/example/android/tflitecamerademo/ImageClassifier.java

```diff
@@ -176,12 +179,12 @@ public void useGpu() {
   }
 
   public void useCPU() {
-    tfliteOptions.setUseNNAPI(false);
     recreateInterpreter();
   }
 
   public void useNNAPI() {
-    tfliteOptions.setUseNNAPI(true);
+    nnapiDelegate = new NnApiDelegate();
+    tfliteOptions.addDelegate(nnapiDelegate);
```

Comment:
Thinking about this some more, we might need to clear the delegates from the options. Otherwise you might still have a GPU delegate (or an NNAPI delegate) when using the CPU.

Feel free to add a "clearDelegates()" method to Interpreter.Options. Alternatively, maybe undo the changes to this file, and we can fix that in a separate change?

Reply:
Argh, thanks, I understand you concern now. I'll revert this, check why I didn't run into the situation you mentioned, and send a separate PR for tflitecamerademo.

Reply:
I further tested the app. It really works because [recreateInterpreter()](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/java/demo/app/src/main/java/com/example/android/tflitecamerademo/ImageClassifier.java#L163-L168) closes current interpreter instance and creates a new instance. And, in [close()](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/java/src/main/java/org/tensorflow/lite/NativeInterpreterWrapper.java#L92-L115) of `java.org.tensorflow.lite.NativeInterpreterWrapper`, delegates are cleared already.

Reply:
Ah good point, looks good then!